### ISSUE TITLE
Suddenly appearing issues distinct from a server down situation

### ISSUE THREAD

**Issue by TehBengYen419** [farnking/Test-Dragon-Forum-of-TBots#41]
In the same scenario, the interface was functioning normally before, but now, upon logging in, it appears as shown in the screenshot, followed by program termination without error
![Screenshot 2024-01-01 101621](https://github.com/farnking/Test-Dragon-Forum-of-TBots/assets/59071923/b08a5dc5-4b6d-43ee-ae67-05f5e794c83d)

**Comment by farnking** [comment]
thank！
Can you cut the output in cmd and send it tome?

王凡

從我的iPhone傳送

Akira ***@***.***> 於 2024年1月1日 上午10:28 寫道：

﻿

In the same scenario, the interface was functioning normally before, but now, upon logging in, it appears as shown in the screenshot, followed by program termination without error
Screenshot.2024-01-01.101621.png (view on web)<https://github.com/farnking/Test-Dragon-Forum-of-TBots/assets/59071923/b08a5dc5-4b6d-43ee-ae67-05f5e794c83d>

**Comment by TehBengYen419** [comment]
It looks like there might be an error here
![Screenshot 2024-01-01 120739](https://github.com/farnking/Test-Dragon-Forum-of-TBots/assets/59071923/62976517-571b-4231-818b-0ead5ea70e27)

The program terminates from here without any error
![Screenshot 2024-01-01 120637](https://github.com/farnking/Test-Dragon-Forum-of-TBots/assets/59071923/2d57612e-4fa9-47fc-ad2d-cf3c7c22a7e0)

**Comment by farnking** [comment]
removed a problem, 
Please try again.

**Comment by TehBengYen419** [comment]
The program requires an update, but it seems to repeatedly ask for updates every time it is reopened. Do I need to download it again, or is there a solution?

**Comment by farnking** [comment]
Yes, you need download again from the web: https://www.test-dragon.com/

The reason usually is that the newest version modified code before the login action.

**Comment by TehBengYen419** [comment]
Thanks for addressing the issue, the server down again QQ

**Comment by farnking** [comment]
I might forget to restart it after modification.
